src/argilla/client/feedback/training/schemas.py

```diff
@@ -324,7 +325,7 @@ def for_reward_modeling(
             ...     else:
             ...         chosen = sample["response-2"]
             ...         rejected = sample["response-1"]
-            ...     return chosen, rejected
+            ...     yield (chosen, rejected)
```

Comment:
You don't actually need the brackets, you can just do `yield chosen, rejected`

Reply:
I know, it was more about mentioning it explicitly

Reply:
Makes sense, it might be clearer for inexperienced programmers